projects/authentication/docs/client-access-token.md

```diff
@@ -16,30 +16,27 @@ If your browser or native application cannot work with user logins via publiq's
 
 See [requesting client credentials](./requesting-credentials.md) how to obtain a set of client credentials.
 
-## Flow
-
-1.  The client makes a request to the authorization server with its id and secret.
-
-2.  The authorization server validates the request and, if successful, sends a response with an access token.
-
-3.  The client can now use the access token to call the API by using the obtained access token as a `Bearer` token in the `Authorization` header.
-
-<!-- theme: warning -->
-
-> ##### Security warnings
->
-> *   ✅ **Always** request a client access token from a **backend** application
-> *   ❌ **Never** use or expose your client access token from a **frontend** application
->
-> If you use or store your client secret or client access token in a frontend application they can be **stolen** and **abused** and your client will get blocked!
->
-> If you need to do API calls from a frontend application, use [client identification](./client-identification.md) or a [user access token](./user-access-token.md) depending on which methods the API endpoint accepts.
-
-> ##### OAuth2
->
-> Client access tokens are requested using the standardized [OAuth 2.0 Client Credentials Grant](https://oauth.net/2/grant-types/client-credentials/). If you are familiar with this flow, you can skip most of the example flow below. Do check the info about the required `audience` property though.
+## How it works
+
+```mermaid
```

Comment:
The diagram is very clear (not blurry at all as mentioned in the description of your pull request) 👍

Reply:
For me the labels of the arrows are a bit "blurry" around the edges of the text.

![Screen Shot 2022-08-19 at 10 57 05](https://user-images.githubusercontent.com/959026/185584426-6ccf1926-4e69-4402-98e0-d8b85e35d087.png)

They look like they're bold, but in fact their font weight is 400 (normal, not bold), but they have a "stroke". That is a feature in SVGs to add a border of a different color around objects and text. It seems to be that stroke makes the text look "bold" but also not as crispy as if it would just use font weight 700.

(With the small screenshot it's maybe not super visible here)

For example if I disable the stroke in my web inspector it already looks a lot better:

![Screen Shot 2022-08-19 at 10 57 01](https://user-images.githubusercontent.com/959026/185600892-1a83cbbf-3c5b-4a5e-ab52-e04ec1e5858f.png)

And when I then add font-weight 700 it looks bold again, but the edge is noticeable less blurry than in my first screenshot:

![Screen Shot 2022-08-19 at 10 58 04](https://user-images.githubusercontent.com/959026/185601050-ba5240c2-53f2-411b-b270-952b586590a3.png)

Reply:
Anyway I found an issue in the mermaid JS library about this so I assume it will be fixed in the future. And it's still readable, just not ideal.